alibi_detect/cd/pytorch/lsdd_online.py

```diff
@@ -178,26 +178,29 @@ def _configure_ref_subset(self):
             h_init = self.c2s - self.k_xtc.mean(0)  # (Eqn 21)
             lsdd_init = h_init[None, :] @ self.H_lam_inv @ h_init[:, None]  # (Eqn 11)
 
-    def score(self, x_t: np.ndarray) -> Union[float, None]:
+    def _update_state(self, x_t: Union[np.ndarray, Any]):
+        self.t += 1
+        x_t = super()._preprocess_xt(x_t)
```

Comment:
Same comment as in the MMD equivalent: would it make sense to do preprocessing of `x_t` separately from `_update_state` and feed the preprocessed `x_t` to `_update_state`?

Reply:
Sure, we can certainly separate these. My thinking behind calling `_preprocess_xt` in `_update_state` was that essentially we are updating state with `x_t`, so it does make some sense to have it inside `_update_state` (as we're only preprocessing `x_t` in order to update the state). But I suppose thinking about more complex handling of state in the future (loading state etc), perhaps it is clearer for us to separate out instead.